core/domain/suggestion_domain_test.py

```diff
@@ -38,11 +41,9 @@ def setUp(self):
         self.signup(self.ASSIGNED_REVIEWER_EMAIL, 'assignedReviewer')
         self.assigned_reviewer_id = self.get_user_id_from_email(
             self.ASSIGNED_REVIEWER_EMAIL)
-
-    def test_to_dict(self):
-        expected_suggestion_dict = {
+        self.suggestion_dict = {
             'suggestion_id': 'exploration.exp1.thread1',
-            'suggestion_type': suggestion_models.SUGGESTION_EDIT_STATE_CONTENT,
+            'suggestion_type': suggestion_models.SUGGESTION_TYPE_EDIT_STATE_CONTENT, # pylint: disable=line-too-long
```

Comment:
Try to remove the pragma:

    'suggestion_type': (
        suggestion_models.SUGGESTION_TYPE_EDIT_STATE_CONTENT)

Reply:
Done